### PR TITLE
round-tracker: add upper limit to max tasks per node

### PR DESCRIPTION
**Pull request by juliangruber**

### CODE REVIEW

File: api/lib/round-tracker.js [hunk 1]
```diff
@@ -264,10 +268,11 @@ export async function maybeCreateSparkRound (pgClient, {
       $2,
       $3,
       $4,
-      (
-        $5::int /* previousRound.max_tasks_per_node || BASELINE_TASKS_PER_NODE */
-        * $6::int /* TASKS_EXECUTED_PER_ROUND */
-        / $7::int /* previousRound.measurement_count || TASKS_EXECUTED_PER_ROUND */
+      LEAST(
+        $5,
```
Comment (bajtos) [inline api/lib/round-tracker.js:272]: Please add a comment to explain that `$5` is for `MAX_TASKS_PER_NODE_LIMIT`

**Comment by juliangruber** [comment]
> With our 20-minute rounds, 100 tasks per node mean one retrieval check every 12 seconds. That feels a bit too much to me. Having said that, we should be hitting this maximum limit very rarely, so let's see if anybody ever complains.

I thought that every 12 seconds shouldn't be too much for a single node. Do you have another default to suggest?

File: api/lib/round-tracker.js (hunk 1 above)
Comment (juliangruber) [reply api/lib/round-tracker.js:272]: https://github.com/filecoin-station/spark-api/commit/58dc884f9f2295edc62ee4ee39beee0bdc6f0ce0

**Comment by bajtos** [comment]
At some point in the past, we used to do one check per minute - that means 20 tasks per node, which feels like a rather low number.

Let's keep the current value and wait for feedback.

**Pull request merged**